site/themes/dojo/layout/_partial/landingPage/hero.ejs

```diff
@@ -0,0 +1,90 @@
+<section class="hero">
+	<div class="shapes">
+		<!--
```

Comment:
Should we remove these comments? If not, make a note as to why they've been kept.